samza-test/src/main/java/org/apache/samza/test/framework/StateAssert.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.samza.test.framework;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.samza.storage.kv.RocksDbKeyValueReader;
+import org.apache.samza.storage.kv.RocksDbTableDescriptor;
+
+import static org.junit.Assert.*;
+
+/**
+ *  Assertion utils on the content of a Table described by
+ *  {@link org.apache.samza.operators.TableDescriptor}.
+ */
+public class StateAssert {
+
+  /**
+   * Verifies that the {@code key} is present in the table described by {@code RocksDbTableDescriptor}
+   * @param key expected key in the table
+   * @param tableDescriptor describes the RocksDb table
+   */
+  public static void contains(Object key, RocksDbTableDescriptor tableDescriptor) {
```

Comment:
Is this necessary when you already have the "get" API?
If you do keep this, can you please make the argument ordering consistent with "get"? Also, can you add a javadoc clarifying how this works across multiple partitions?